types/three/three-core.d.ts

```diff
@@ -6440,17 +6551,37 @@ export class EdgesGeometry extends BufferGeometry {
     constructor(geometry: BufferGeometry, thresholdAngle: number);
 }
 
+export interface ExtrudeGeometryParameters {
+    curveSegments?: number;
+    steps?: number;
+    amount?: number;
+    bevelEnabled?: boolean;
+    bevelThickness?: number;
+    bevelSize?: number;
+    bevelSegments?: number;
+    extrudePath?: CurvePath<Vector>;
+    frames?: any;
+    UVGenerator?: any;
```

Comment:
It needs to match this: https://github.com/mrdoob/three.js/blob/61520f0d8a935f599e658119d7020e0525e17f35/src/geometries/ExtrudeGeometry.js#L756